hnn_core/viz.py

```diff
@@ -182,6 +178,7 @@ def plot_laminar_lfp(times, data, contact_labels, tmin=None, tmax=None,
             col = color
         ax.plot(plot_times, plot_data + trace_offsets[contact_no],
                 label=f'C{contact_no}', color=col)
+        ax.set_xlim(right=plot_times[-1])
```

Comment:
```suggestion
        ax.set_xlim(right=times[-1])
```
`xlim` should always end at the same point regardless of whether or not the time series was decimated.